editor/blocks/quote-block/index.js

```diff
@@ -0,0 +1,23 @@
+const { query, html, text } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'wp/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: query( 'cite,footer', text() )
+	},
+
+	edit( attributes, onChange ) {
+		return wp.element.createElement( wp.blocks.EditableQuote, {
+			value: attributes.value,
+			citation: attributes.citation,
+			onChange: ( value ) => onChange( { value } )
+		} );
+	},
+
+	save( attributes ) {
+		return wp.element.createElement( 'p', null, attributes.value );
```

Comment:
I think the current idea is to use `renderToString` outside the block (in a global maner). Relevant to this discussion https://github.com/WordPress/gutenberg/pull/284#pullrequestreview-27450704 
https://github.com/WordPress/gutenberg/pull/302#discussion_r107147633